### PR TITLE
Uploaded Nobu Casa Threat Model

### LINKED ISSUE #51: DFD 2 + Report

### LINKED ISSUE #56: DFD 2 Summary Of Observations

### PR DESCRIPTION
**Pull request by CernickyR**

### CODE REVIEW

**Review by RocketSockz — APPROVED** [review]
Nice work on this man.

**Comment by Chrs987** [comment]
Everything looks good and the writeup and threats you addressed were well thought out.

File: Threat Modeling.md [hunk 1]
```diff
@@ -51,7 +53,7 @@ Home Assistant gives users the ability to view the system logs in several ways.
 
 
 ### Tampering
-
+Home Assistant gives the user the ability to have mechanisms in place to help mitigate Tampering of the system. Following the [securing documentation](https://www.home-assistant.io/docs/configuration/securing/) provides the user with a secure platform of Home Assistant. The two main forms of tampering that Home Assistnat is prone to are Physical and Digital. Physical Tampering should be mitigated by the user. Digital Tampering to the system is mitigated by only having accounts with root access able to edit config files on the system. Some risk that does apply to the applicaiton are from community add-ons. The community add-ons that are advertised have been community vetted [Community Add-Ons ](https://community.home-assistant.io/tags/hassio-repository).
```
Comment (Jbibianochavez) [inline Threat Modeling.md:56]: I would take a look at these sentences "Digital Tampering to the system is mitigated by only having accounts with root access able to edit config files on the system. Some risk that does apply to the application are from community add-ons. The community add-ons that are advertised have been community vetted [Community Add-Ons ]" 
Possibly change them to something like sentences below. Everything else looks good to go!
'Digital tampering is mitigated by only giving accounts with root access the ability to edit config files on the system. 
Some risk that does apply to the application are from  [community add-ons] (https://community.home-assistant.io/tags/hassio-repository). Add-ons that are advertised have been vetted by the community. '

**Review by Jbibianochavez — APPROVED** [review]
Made a slight suggestion on grammar, other than that, everything looks good to merge.

**Pull request merged**